go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	golang.org/x/mod v0.11.0
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.1
-	sigs.k8s.io/controller-runtime v0.15.0
+	sigs.k8s.io/controller-runtime v0.14.7
```

Comment:
Are you sure we want to move down? https://github.com/kubernetes-sigs/controller-runtime/commits/v0.15.3 suggests golang.org/x/net has been upgraded to v0.17.0

Reply:
I am ok with moving down to 0.14.7.. just that I want to confirm if we move higher to v0.15.3, we are not forcing HAS and/or devfile to upgrade as well..

Reply:
@maysunfaisal we had already added, after a community call, the https://github.com/devfile/api/issues/1275 to align the controller runtime with the version that HAS is using. In the past we had to create different releases for HAS in order to use `v0.14.4` instead of `v0.15.0`.